torchvision/ops/boxes.py

```diff
@@ -154,9 +154,6 @@ def box_convert(boxes: Tensor, in_fmt: str, out_fmt: str) -> Tensor:
     Returns:
         boxes (Tensor[N, 4]): Boxes into converted format.
     """
-    allowed_fmts = ("xyxy", "xywh", "cxcywh")
-    assert in_fmt in allowed_fmts
-    assert out_fmt in allowed_fmts
```

Comment:
Is this not supported by torchscript?
I just tried the following and it worked:
```python
In [1]: import torch

In [2]: def f(x: str):
   ...:     L = ('a', 'b')
   ...:     assert x in L
   ...:     return x
   ...:

In [3]: ff = torch.jit.script(f)

In [4]: print(ff.code)
def f(x: str) -> str:
  if torch.__contains__(["a", "b"], x):
    pass
  else:
    ops.prim.RaiseException("AssertionError: ")
  return x
```

Reply:
I just asked @vfdev-5 and removed it. We removed it since then tests will not trigger value error as assert would stop this. I added tests to check `ValueError` too. 😕 I am not sure if I should add it back.

Reply:
If `assert` is not good, then let's change instead to `ValueError`, something like
```python
if in_fmt not in allowed_fmts:
    raise ValueError(...)
```
Having those checks is important for better user experience, otherwise their code could silently fail